### PR TITLE
Feature footpaths ldfetch

### PR DESCRIPTION
**Pull request by DylanVanAssche**
# What?

This PR adds footpath support to client.js With this feature you can add nearby stations as possible connections while calculating the earliest arrival time.

This PR is required to allow https://github.com/linkedconnections/csa.js/pull/27 to function properly and fixes the to do from PR #21 

# Environment

- Node 8.11.3 LTS
- OpenSUSE Leap 15 x64

### CODE REVIEW

File: bin/client.js [hunk 1]
```diff
@@ -23,36 +23,51 @@ program
   })
   .parse(process.argv);
 
-var configFile = program.config || path.join(__dirname, '../config.json'),
-    config = JSON.parse(fs.readFileSync(configFile, { encoding: 'utf8' }))
+var configFile = program.config || path.join(__dirname, '../config-example.json'),
```
Comment (pietercolpaert) [inline bin/client.js]: By default, we want the config.json to be chosen... No?

File: bin/client.js [hunk 2]
```diff
@@ -61,25 +76,34 @@ client.query(q, function (stream, source, connectionsStream) {
     count++;
   });
   stream.on('result', function (path) {
-    path.forEach(function (connection) {
-      console.log(connection.departureTime.toISOString() + " at " + connection.departureStop + " To arrive in " + connection.arrivalStop + " at " +  connection.arrivalTime.toISOString());
+    paths.push(path);
+    console.log(path);
```
Comment (pietercolpaert) [inline bin/client.js]: Shouldn’t be here probably?

File: bin/client.js (hunk 1 above)
Comment (DylanVanAssche) [reply bin/client.js]: Yes indeed, I will fix it

File: lib/lc-client.js [hunk 3]
```diff
@@ -1,16 +1,21 @@
-var Planner = require('csa').BasicCSA,
-    LDFetch = require('ldfetch'),
-    Fetcher = require('./Fetcher');
+var Planner = require('../../csa.js/lib/csa').BasicCSA,
+  TimespanPlanner = require('../../csa.js/lib/csa').TimespanCSA,
```
Comment (julianrojas87) [inline lib/lc-client.js]: Dependencies should be imported from node_modules

**Comment by DylanVanAssche** [comment]
I can't resolve the merge conflicts myself due missing write access in this repository.

**Comment by julianrojas87** [comment]
I am doing that don't worry :)

**Pull request merged**